### ISSUE TITLE
dev20 - error in spigot 1.9.4

### ISSUE THREAD

**Issue by cwken10** [bensku/Skript#191]
after start server, it spam these lines of code only, then skript totally cannot use
```
[02:01:51] [Server thread/INFO]: [Skript] Enabling Skript v2.2-dev20
[02:01:51] [Server thread/ERROR]: Error occurred while enabling Skript v2.2-dev20 (Is it up to date?)
java.lang.NoSuchMethodError: org.bukkit.entity.Villager$Profession.isZombie()Z
	at ch.njol.skript.entity.VillagerData.<clinit>(VillagerData.java:59) ~[?:?]
	at java.lang.Class.forName0(Native Method) ~[?:1.8.0_91]
	at java.lang.Class.forName(Class.java:348) ~[?:1.8.0_91]
	at ch.njol.skript.SkriptAddon.loadClasses(SkriptAddon.java:110) ~[?:?]
	at ch.njol.skript.Skript.onEnable(Skript.java:286) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:292) ~[server.jar:git-Spigot-c6871e2-0cd0397]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:340) [server.jar:git-Spigot-c6871e2-0cd0397]
	at com.massivecraft.massivelag.inject.event.MassiveLagPluginLoader.enablePlugin(MassiveLagPluginLoader.java:216) [MassiveLag%202.8.20.jar:?]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405) [server.jar:git-Spigot-c6871e2-0cd0397]
	at org.bukkit.craftbukkit.v1_9_R2.CraftServer.loadPlugin(CraftServer.java:362) [server.jar:git-Spigot-c6871e2-0cd0397]
	at org.bukkit.craftbukkit.v1_9_R2.CraftServer.enablePlugins(CraftServer.java:322) [server.jar:git-Spigot-c6871e2-0cd0397]
	at net.minecraft.server.v1_9_R2.MinecraftServer.t(MinecraftServer.java:416) [server.jar:git-Spigot-c6871e2-0cd0397]
	at net.minecraft.server.v1_9_R2.MinecraftServer.l(MinecraftServer.java:381) [server.jar:git-Spigot-c6871e2-0cd0397]
	at net.minecraft.server.v1_9_R2.MinecraftServer.a(MinecraftServer.java:336) [server.jar:git-Spigot-c6871e2-0cd0397]
	at net.minecraft.server.v1_9_R2.DedicatedServer.init(DedicatedServer.java:268) [server.jar:git-Spigot-c6871e2-0cd0397]
	at net.minecraft.server.v1_9_R2.MinecraftServer.run(MinecraftServer.java:532) [server.jar:git-Spigot-c6871e2-0cd0397]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_91]
```

**Comment by bensku** [comment]
Oops, this broke since I fixed this for 1.10. Since 1.9 is also supported, fix will come eventually.

**Comment by JKGamerxD** [comment]
Waiting for this fix very soon, it breaks anything at 1.9.

**Comment by bensku** [comment]
Should be fixed now. Please report if it isn't, didn't have 1.9 server available to test it.
